amundsen_application/static/js/components/TableDetail/ColumnListItem/ColumnType/styles.scss

```diff
@@ -56,13 +54,12 @@ $modal-dialog-width: 418px;
     /* Override react-bootstrap styles to match design */
     padding: $spacer-3 !important;
 
-    /* A design audit of modals is needed to reconcile differences accumulated over time */
-    .modal-title {
-      font-family: $font-family-body !important;
+    .main-title {
+      @extend %text-title-w1;
     }
-
-    .column-name {
-      font-size: $w2-font-size;
+    .sub-title {
+      @extend %text-subtitle-w3;
+      color: $text-secondary;
```

Comment:
Nit: Not super sure, but usually linters warn if we don't let a space after extends or mixins.

Reply:
Since we still have so many I haven't been noticing if my code has increased those warnings, let me check.

Reply:
I didn't see a warning on this, but fixed some in these files while I was looking.